modules/command-engine/core/src/main/scala/surge/internal/persistence/PersistentActor.scala

```diff
@@ -47,8 +47,11 @@ object PersistentActor {
       @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "aggregateType", visible = true) aggregateState: Option[S])
       extends JacksonSerializable
 
-  sealed trait ACK extends ActorMessage with JacksonSerializable
+  sealed trait ACK
+      extends ActorMessage
+      with JacksonSerializable
 
+      // FIXME This is an explicit response type now
```

Comment:
Need some renaming as this is no longer necessarily `aggregateState` but a generic `response`

Reply:
Will revisit this as part of the PersistentActor cleanup later as well...